scipy/linalg/misc.py

```diff
@@ -62,23 +65,25 @@ def norm(a, ord=None, axis=None, keepdims=False, check_finite=True):
     =====  ============================  ==========================
     None   Frobenius norm                2-norm
     'fro'  Frobenius norm                --
-    inf    max(sum(abs(x), axis=1))      max(abs(x))
-    -inf   min(sum(abs(x), axis=1))      min(abs(x))
-    0      --                            sum(x != 0)
-    1      max(sum(abs(x), axis=0))      as below
-    -1     min(sum(abs(x), axis=0))      as below
+    'nuc'  nuclear norm                  --
+    inf    max(sum(abs(a), axis=1))      max(abs(a))
+    -inf   min(sum(abs(a), axis=1))      min(abs(a))
+    0      --                            sum(a != 0)
+    1      max(sum(abs(a), axis=0))      as below
+    -1     min(sum(abs(a), axis=0))      as below
     2      2-norm (largest sing. value)  as below
     -2     smallest singular value       as below
-    other  --                            sum(abs(x)**ord)**(1./ord)
+    other  --                            sum(abs(a)**ord)**(1./ord)
     =====  ============================  ==========================
 
     The Frobenius norm is given by [1]_:
 
         :math:`||A||_F = [\\sum_{i,j} abs(a_{i,j})^2]^{1/2}`
 
-    The ``axis`` and ``keepdims`` arguments are passed directly to
-    ``numpy.linalg.norm`` and are only usable if they are supported
-    by the version of numpy in use.
+    The nuclear norm is the sum of the singular values.
+
+    Both the Frobenius and nuclear norm orders are only defined for
+    matrices and raise a ValueError when `a.ndim != 2`.
```

Comment:
For these norms, `a.ndim > 2` is allowed when used in combination with `axis`, so this statement isn't strictly true.  We could just drop the second part of the sentence:
```suggestion
    matrices.
```